test/sagemaker_tests/tensorflow/tensorflow2_training/requirements.txt

```diff
@@ -23,5 +23,4 @@ requests-mock
 tensorflow
 awscli
 fabric==2.5.0
-sagemaker-experiments==0.1.7
```

Comment:
Why remove this?

Reply:
sagemaker-experiments needs to be installed only for py3 images. so I moved this to expliclty install for py3 images to sagemaker.py